lua/templar.lua

```diff
@@ -83,10 +83,13 @@ local function source()
 end
 
 -- registers a new file extension to use the template with
-local function register(filename)
+local function register(filename, alias)
     -- Generate template path from filename
     -- This is basically replacing each * in the filename by template
     local temppath = 'templates/' .. string.gsub(filename, "%*", "template")
+    if alias then
+	temppath = 'templates/'.. alias .. '.' .. vim.fn.fnamemodify(temppath, ':e') 
```

Comment:
Hmmm... I'd suggest to move this when we actually compute `temppath`, to correctly compute the path first try, instead of changing it afterwards.
That'll improve readability too.